.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+
+name: Build and Test
+
+on:
+  push:
+    branches: [ "master", "develop"]
+  pull_request:
+    branches: [ "master", "develop" ]
```

Comment:
We could improve this a bit so that the build happens only if paths that have code changes are affected that we need to test. For example in X-Road we have:
```yaml
on: 
  # Capture this event so that gradle caches are updated when a PR is merged to develop
  # More information on why: https://github.com/gradle/gradle-build-action#using-the-caches-read-only
  push:
    branches:
      - develop
    paths:
      - 'src/**'
      - '.github/**'
      - 'ansible/**'
      - 'Docker/**'
  pull_request:
    types: [opened, synchronize, reopened]
    paths:
      - 'src/**'
      - '.github/**'
      - 'ansible/**'
      - 'Docker/**'
```

I think we could also use PR types, instead of the branches for `pull_request`.